.github/workflows/build.yml

```
@@ -54,7 +54,9 @@ jobs:
         uses: actions/upload-artifact@v2
         with:
           name: ${{ github.sha }}
-          path: ./*/build/distributions/*.zip
+          path: | 
+            ./*/build/distributions/*.zip
+            ./sentry-android-ndk/build/intermediates/cmake/release
```

```suggestion
            ./sentry-android-ndk/build/intermediates/merged_native_libs/release/out/lib
```

used `sentry-cli difutil check` and they are the same files, but I'd rather chose the `merged_native_libs` because if there are files not built by the solution, they will make into the `merged_native_libs` as well